anta/tests/interfaces.py

```diff
@@ -230,29 +235,15 @@ class Input(AntaTest.Input):
         interfaces: list[InterfaceState]
         """List of interfaces with their expected state."""
 
```

Comment:
Here we need to keep a class variable to maintain the current behavior:
`InterfaceState: ClassVar[type[InterfaceState]] = InterfaceState`

Reply:
Added.